com.github.akiraux.akira.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.akiraux.akira
+runtime: io.elementary.Platform
+runtime-version: '6'
+sdk: io.elementary.Sdk
+command: com.github.akiraux.akira
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # needed for perfers-color-scheme
+  - '--system-talk-name=org.freedesktop.Accounts'
```

Comment:
Is it really needed? I thought elementary added a gsetting proxy for it.

Reply:
It's needed based on their documentation to detect the preferred color scheme.
Do you see any issue with this?

Reply:
Not really no, I just thought something has changed meanwhile. I don't think it's a good idea to open a hole in the sandbox for each app just to get color scheme/dark mode working but that's not related to Akira.